package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@mdn/translated-content",
+  "version": "1.0.0",
+  "description": "Translated content of https://developer.mozilla.org",
+  "private": true,
+  "engines": {
+    "node": "^16.0.0 || >=18.0.0"
+  },
+  "scripts": {
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mdn/translated-content.git"
+  },
+  "author": "MDN Web Docs",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "bugs": {
+    "url": "https://github.com/mdn/translated-content/issues"
+  },
+  "homepage": "https://github.com/mdn/translated-content#readme",
+  "devDependencies": {
+  }
```

Comment:
Could we reorder the fields, loosely based on the TOC of the [npm docs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json)?

- name
- version
- description
- private
- homepage
- bugs
- license
- author
- repository
- scripts
- dependencies
- devDependencies
- engines

Reply:
Done, but I didn't add the dependicies stub, since I'm not sure it will be used here